pkg/util/version/version.go

```diff
@@ -58,3 +63,26 @@ func (v *Version) Lt(w *Version) bool {
 
 	return false
 }
+
+func (v *Version) Eq(w *Version) bool {
+	return bytes.Compare(v.V[:], w.V[:]) == 0
+}
+
+func GetUpgrade(v *Version) (*Upgrade, error) {
+	for _, u := range Upgrades {
+		if u.Version.V[0] == v.V[0] &&
+			u.Version.V[1] == v.V[1] {
+			return &u, nil
+		}
+	}
+	return nil, fmt.Errorf("upgrade for %d not found", v.V[:])
```

Comment:
I'm not sure this should be an error case - that might prevent us upgrading clusters when 4.5 comes out and someone upgrades to it early?  Perhaps the contract should be that nil is returned?

Reply:
We can do either way. It will be logged by `action` calling or we log it here. The challenge is that I want this to be propagated in the logs (so we can see what happened if this happens). But This would imply I have to pass logger into version. And Im not sure if this is right idea.